src/Microsoft.Health.Fhir.CosmosDb/Features/Search/Queries/QueryBuilder.cs

```diff
@@ -137,23 +134,12 @@ private void AppendFilterCondition(string name, object value)
                         .AppendLine(_queryParameterManager.AddOrGetParameterMapping(value));
             }
 
-            private void AppendSystemDataFilter(string keyword = null)
+            private void AppendSystemDataFilter()
             {
-                // Ensure that we exclude system metadata
-
-                if (!string.IsNullOrEmpty(keyword))
-                {
-                    _queryBuilder.Append(keyword).Append(" ");
-                }
-
                 _queryBuilder
-                    .Append("(")
-                    .Append("IS_DEFINED(").Append(SearchValueConstants.RootAliasName).Append(".isSystem)")
-                    .Append(" = ").Append(_queryParameterManager.AddOrGetParameterMapping(false))
-                    .Append(" OR ")
+                    .Append(" WHERE ")
                     .Append(SearchValueConstants.RootAliasName).Append(".isSystem")
-                    .Append(" = ").Append(_queryParameterManager.AddOrGetParameterMapping(false))
-                    .AppendLine(")");
+                    .Append(" = ").Append(_queryParameterManager.AddOrGetParameterMapping(false));
```

Comment:
you should make the send `Append` into AppendLine.

Reply:
Sure. I will update the PR.